packages/components/src/components/video/video.stories.ts

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+import '../../solid-components';
+import { html } from 'lit-html';
+import { storybookDefaults, storybookTemplate } from '../../../scripts/storybook/helper';
+import { withActions } from '@storybook/addon-actions/decorator';
+
+const { argTypes, args, parameters } = storybookDefaults('sd-video');
+const { generateTemplate } = storybookTemplate('sd-video');
+
+/**
+ * The `sd-video` component can be used to wrap external video elements in order to provide basic styling for Union Investment.
+ * ## Implementation
+ * Use in combination with a viewer component (e. g. from Moving Image) or a bare `<video>`-Tag in addition to `sd-media`.
+ * `<sd-media>
```

Comment:
should we add some information here about the non-existing sd-media component? Or maybe leave it out of here and add it as soon as the component is ready.

Reply:
+1 – would remove everything related `sd-media`